src/ralph/cli.py

```diff
@@ -153,23 +153,53 @@ def wrapper(command):
     required=True,
     help="Container format parser used to extract events",
 )
+def extract(parser):
+    """Extract input events from a container format using a dedicated parser."""
+
+    logger.info("Extracting events using the %s parser", parser)
+
+    parser = get_class_from_name(parser, PARSERS)()
+
+    for event in parser.parse(sys.stdin):
+        click.echo(event)
+
+
+@cli.command()
 @click.option(
-    "-c",
-    "--chunksize",
-    type=int,
-    default=DEFAULT_GELF_PARSER_CHUNCK_SIZE,
```

Comment:
I removed here the `chunksize` option of the `extract` command as it's not used anymore, is it ok ?